.github/workflows/ci.yaml

```
@@ -47,8 +52,30 @@ jobs:
       - name: Build
         run: pnpm turbo run build
 
-      - name: Format check
-        run: pnpm turbo run format:check
+      - name: Format code
+        run: pnpm turbo run format
+
+      - name: Commit formatting changes if needed
+        run: |
```

actionlint (1.7.4)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)